wiki/Beatmap_Editor/Beat_Snap_Divisor/en.md

```diff
@@ -1,19 +1,60 @@
+Beat snap divisor
+==================
+
 The Beat Snap Divisor is used to limit where notes can be placed so that they always fall on the beat. Moving the slider to the right increases the granularity of the notes, that is, more notes can be placed in the same measure.
 
-Each of the settings put different colors of ticks on the [Timelines](Timelines "wikilink"). The ticks are as follows (for [Standard](Standard "wikilink"), [Taiko](Taiko "wikilink"), and [Catch the Beat](Catch_the_Beat "wikilink")):
+Each of the settings put different colors of ticks on the [Timelines](/wiki/Beatmap_Editor/Timelines). The ticks are as follows (for [Standard](/wiki/Game_Modes/osu!), [Taiko](/wiki/Game_Modes/osu!taiko), and [Catch the Beat](/wiki/Game_Modes/osu!catch)):
+
+### Divisor 1/1
+
+![](/wiki/shared/BSD_1_1b.jpg)
+
+- Full Beats
+- White ticks only
+- Best used for Easy difficulty.
+
+### Divisor 1/2
+
+![](/wiki/shared/BSD_1_2.jpg)
```

Comment:
L18: Add text on ``[]`` to phrase which supposed to be there if the image doesn't load